cli/activate.go

```diff
@@ -98,7 +98,7 @@ func Activate(stepLibURI, id, version, destination, destinationStepYML string, u
 				versionList = versionList + fmt.Sprintf("\n- %s", version)
 			}
 
-			errMsg := fmt.Sprintf("version is not available in the local cache and $BITRISE_BETA_OFFLINE_MODE is set. %s", versionList)
+			errMsg := fmt.Sprintf("version is not available in the local cache and $BITRISE_OFFLINE_MODE is set. %s", versionList)
```

Comment:
Spotted this earlier. We can't import the env key from the CLI codebase because that would be a circular dependency